src/components/UI/Text/TimeInput.vue

```diff
@@ -184,6 +177,16 @@ export default {
         this.localMillisecond = this.timeObject.millisecond;
       }
     },
+    timeObject: {
+      handler() {
+        // update the local values if the user selects a different item than the current selected one
+        this.localHour = this.timeObject.hour;
+        this.localMinute = this.timeObject.minute;
+        this.localSecond = this.timeObject.second;
+        this.localMillisecond = this.timeObject.millisecond;
+      },
+      deep: true,
+    },
```

Comment:
why are these properties not computed properties?

Reply:
We want to store user ka locally entered value in these variables
If we make it computed toh everytime `timeObject` changes, they will be recalculated